src/patternfly/components/Tabs/tabs.scss

```diff
@@ -0,0 +1,188 @@
+@import "../../patternfly-utilities";
+
+.pf-c-tabs-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+// Tabs
+.pf-c-tabs {
+  // Primary Tab Items
+  --pf-c-tabs__item--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-tabs__item--current--Color: var(--pf-global--active-color--100);
```

Comment:
```suggestion
  --pf-c-tabs__item--m-current--Color: var(--pf-global--active-color--100);
```

Reply:
just update the rest of the scss to use `m-current` in the var name since it's a modifier class

Reply:
Added the `m-` to `--pf-c-tabs__item--m-current--Color`, `--pf-c-tabs__item--m-current--BoxShadow`, and `--pf-c-tabs__item--m-current--ZIndex`. Thanks!